<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>663</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Candara,Calibri,loma</family>
    <pointsize>14</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Read device parameters</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	background: rgba(32, 80, 96, 100);
	font-family:Candara, Calibri, loma;
	font-size: 14pt;
	font-style: normal;
}

QHeaderView::section{
spacing: 64px;
background: rgba(32, 80, 96, 100);
min-height: 40px;
min-width: 10em;
color: white;
/*border: 2px solid #2e4053;*/
margin: 1px;
text-align: center;
font-family: URW Palladio L, Candara;
font: bold 14pt;
}

#search_frame, #button_frame{
	background: transparent;
	font-family: URW Palladio L, Candara;
	font-size: 14pt
}

#treeView{
	background-color: white;
	font-family: URW Palladio L, Candara;
     font: bold 14pt;
}

QPushButton {
     background-color: rgb(230, 230, 255); /*#4D4D4D;*/
		font-family: URW Palladio L, Candara;
     border-style:outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14pt;
     min-width: 7em;
     padding: 4px;
}

#console_pte{
 	color : green; 
	background-color : rgb(24, 24, 24);
 	font-weight: 14pt;
	font-family: loma;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="connect_frame">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>52</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: white;
	margin: 1px;
	text-align: center;
	font-family: Candara,  URW Palladio L;
	font: bold 14pt;
}

QLineEdit, QComboBox {
    color: rgb(39, 101, 206);
    background-color: rgb(255, 255, 255);
    selection-color: white;
    selection-background-color: rgb(191, 31, 127);
    border: 2px groove gray;
	font-family: Candara,  URW Palladio L;
	font: bold 12pt;
    padding: 2px 4px;
}



QCheckBox::indicator {
	/*border: 3px solid #5A5A5A;
    border : 1px solid #a6a6a6;

    border-radius : 3px;
    top : 0px;*/
    width :36px;
    height : 30px;
}

QCheckBox{ 
	background: white;
	/*background: transparent;
	border: 0;*/
	/*border: 1px solid #5A5A5A;*/
	border-radius : 3px;
    width: 1px;
    height: 30px;
}

QCheckBox:hover, QCheckBox:checked {
    color: orange;
}

QCheckBox:!checked {
    color: blue;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="port_label">
         <property name="font">
          <font>
           <family>Candara,URW Palladio L</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Port name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portname_led">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>ttyS9</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="baudrate_label">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>38</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Candara,URW Palladio L</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Baud rate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="baudrate_cbx">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="parity_label">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Candara,URW Palladio L</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="parity_cbx">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="carriage_return_ckbx">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>4</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="deconnect_button">
         <property name="minimumSize">
          <size>
           <width>187</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>156</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>DISCONNECT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connect_button">
         <property name="minimumSize">
          <size>
           <width>187</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>156</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>CONNECT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPlainTextEdit" name="console_pte">
        <property name="font">
         <font>
          <family>loma</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="small_console_pte">
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>420</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Loma</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="button_frame">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>52</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>341</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="clear_button">
         <property name="minimumSize">
          <size>
           <width>187</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>156</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>CLEAR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quit_button">
         <property name="minimumSize">
          <size>
           <width>187</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>156</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>QUIT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1128</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
